routes.php

```diff
@@ -1,50 +1,24 @@
 <?php
 
 use RainLab\Translate\Models\Message;
-use RainLab\Translate\Classes\Translator;
 
 /*
  * Adds a custom route to check for the locale prefix.
  */
 App::before(function($request) {
-
-    if (App::runningInBackend()) {
-        return;
-    }
-
-    $translator = Translator::instance();
-
-    if (!$translator->isConfigured()) {
-        return;
-    }
-
-    if (!$translator->loadLocaleFromRequest()) {
-        $translator->loadLocaleFromSession();
-        return;
-    }
-
-    if (!$locale = $translator->getLocale()) {
-        return;
-    }
-
```

Comment:
This logic needed just for declare only one `Route`.
I don't care about it and create `Routes` for all locales in the `cms.beforeRoute` event.

So i set needed locale with this code

 ```
$translator = Translator::instance();

        if( !$translator->loadLocaleFromRequest() ) {
            $translator->loadLocaleFromSession();
        }
```
in `Global Middleware` — `RainLab\Translate\Middleware\Locale`